modules/transformer.py

```diff
@@ -94,6 +94,19 @@ def create_html_bracket(bracket_map):
     west = [bracket_map["Western Conference First Round"]]
     east = [bracket_map["Eastern Conference First Round"]]
     finals = []
+    playoff_map = {}
+
+    if not west[0] or not east[0]: # if returned scraped data is empty, need to return an empty map with correct fields
+        empty_map = {
+        "wc1" : [[["",""],["",""]], [["",""],["",""]], [["",""],["",""]], [["",""],["",""]]],
```

Comment:
@MishalAshraf can you change the inner entries to tuples. Let's try to keep it in sync with the other entries in the `dict`. See line 131 as an example.

So `["",""],["",""]` should be `("",""),("","")` for all entries in the map